<launch>  
  <arg name="robot_name" default="pioneer"/>
  <node pkg="mbf_costmap_nav" type="mbf_costmap_nav" respawn="false" name="$(arg robot_name)_move_base_flex" output="screen">
    <param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS"/>
    <param name="tf_timeout" value="1.5"/>
    <param name="robot_frame" value="$(arg robot_name)/base_link"/>
    <param name="planner_max_retries" value="3"/>
    <remap from="frontscan_filtered" to="$(arg robot_name)/frontscan_filtered"/>
    <remap from="cmd_vel" to="$(arg robot_name)/cmd_vel"/>
    <remap from="odom" to="$(arg robot_name)/odom"/>
    <rosparam file="$(find uml_hri_nerve_navigation)/resources/config/$(arg robot_name)_mbf/planners.yaml" command="load" />
    <rosparam file="$(find uml_hri_nerve_navigation)/resources/config/$(arg robot_name)_mbf/base_local_planner_params.yaml" command="load" />
    <rosparam file="$(find uml_hri_nerve_navigation)/resources/config/$(arg robot_name)_mbf/recovery_behaviors.yaml" command="load" />
    <rosparam file="$(find uml_hri_nerve_navigation)/resources/config/$(arg robot_name)_mbf/costmap_common_params.yaml" command="load" ns="global_costmap"/>
    <rosparam file="$(find uml_hri_nerve_navigation)/resources/config/$(arg robot_name)_mbf/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find uml_hri_nerve_navigation)/resources/config/$(arg robot_name)_mbf/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find uml_hri_nerve_navigation)/resources/config/$(arg robot_name)_mbf/local_costmap_params.yaml" command="load" />
  </node>
</launch>